lib/main.js

```diff
@@ -36,6 +37,13 @@ class GolangLanguageClient extends AutoLanguageClient {
         });
     }
 
+    provideAutocomplete() {
+        const config = atom.config.get(PACKAGE_NAME);
+        return Object.assign(super.provideAutocomplete(), {
+            suggestionPriority: config.goplsAutocompleteResultsFirst ? 5 : 1
+        });
```

Comment:
Can you add a comment explaining why this is here. Example "this is to prioritize gopls results over snippets".

Reply:
I have commented it, Thanks for your advice.